roles/kubernetes-apps/policy_controller/calico/templates/calico-kube-controllers.yml.j2

```
@@ -1,15 +1,18 @@
-apiVersion: apps/v1beta2
+apiVersion: extensions/v1beta1
```

I don't know the exact differences between the two versions but instead of using extensions api I guess we should migrate to apps/v1

---

```suggestion
apiVersion: apps/v1
```
OK, The extensions could be compatible old version. And there can be use the apps/v1 replace it.